scripts/util/constants.ts

```diff
@@ -9,6 +9,13 @@ export const DELEGATION_REGISTRY_ADDRESSES = {
   mainnet: "0x00000000000076A84feF008CDAbe6409d2FE638B",
 };
 
+// BytecodeStorageReader library addresses on supported networks
+export const BYTECODE_STORAGE_READER_LIBRARY_ADDRESSES = {
+  // note: same address for goerli and mainnet
```

Comment:
should this say different addresses? this is a case where a create3 pattern might be ideal to get the same address across many different chains, but I don't want to spend time over-optimizing, so totally up to you.

Reply:
AH, good catch. We should indeed. I don't think its worth the headache of doing the create3 pattern here given that this is a stateless read-only library, but if we got that feedback from clients could use create3 pattern for a future version deployment. Updated the documentation! 🙏